openassessment/xblock/self_assessment_mixin.py

```diff
@@ -54,12 +54,14 @@ def self_path_and_context(self):
         context = {'allow_latex': self.allow_latex}
```

Comment:
Set the `time_zone` in the `context` at the time the `context` is initially created here - so it doesn't need to be added in each individual case within this method.  Update other cases in the PR where this happens.